src/routes/ExercisesScreens.tsx

```diff
@@ -26,16 +32,21 @@ interface ExercisesScreenProps {
 const ExercisesScreen = ({ route, navigation }: ExercisesScreenProps): JSX.Element => {
   const { discipline } = route.params
   const { title } = discipline
+  const [isModalVisible, setIsModalVisible] = useState<boolean>(false)
 
   const Header = <Title title={title} description={childrenDescription(discipline)} />
-
+  const currentLevel = 4 //  LUN-131 logic
   const handleNavigation = (item: Exercise): void => {
-    if (item.title === labels.exercises.wordChoice.title && discipline.numberOfChildren < MIN_WORDS) {
-      Alert.alert(labels.exercises.wordChoice.errorWrongModuleSize)
+    if (item.level <= currentLevel) {
```

Comment:
No I think it is fine with `<=`. Current level is the level the user is currently on, so if finishing level 2, the currentLevel is 2, so it would be valid the user can start the exercise of level 2 or am I missing something here?